astropy/cosmology/_io/html.py

```diff
@@ -327,7 +344,9 @@ def write_html_table(
     table.write(file, overwrite=overwrite, format="ascii.html", **kwargs)
 
 
-def html_identify(origin, filepath, fileobj, *args, **kwargs):
+def html_identify(
+    origin: Any, filepath: Any, fileobj: Any, *args: Any, **kwargs: Any
```

Comment:
Why are there so many parameters that are not used?

Reply:
This is how the universal I/O identify functions are written. I suppose some formats need them.

Reply:
As I already mentioned elsewhere, if we need these arguments but we don't really care about their type then we should be annotating them as `object` instead of `Any`. The way I interpret `Any` is that not all types are valid, but the valid types are too difficult to write down because of dynamical code or simply not worth the effort, whereas `object` means that we know for sure that any type is valid.